packages/engine/src/transform/systems/TransformSystem.ts

```diff
@@ -275,7 +280,12 @@ export default async function TransformSystem(world: World) {
       if (makeDirty) world.dirtyTransforms[entity] = true
     }
 
-    const dirtyRigidbodyEntities = invCleanDynamicRigidbodyEntities.filter(isDirty)
+    const dirtyRigidbodyEntities = invCleanDynamicRigidbodyEntities.filter(isDirty).filter(
```

Comment:
Let's create a new filter function, rather filtering twice and creating an arrow function.

Reply:
Sure